roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -14,18 +14,18 @@ kiali_defaults:
   version: "default"
 
   additional_display_details:
-  - title: "API Documentation"
-    annotation: "kiali.io/api-spec"
-    icon_annotation: "kiali.io/api-type"
+    - title: "API Documentation"
+      annotation: "kiali.io/api-spec"
+      icon_annotation: "kiali.io/api-type"
```

Comment:
Something reformated with the wrong indentation. All operator yamls do not "double-indent". All array list items have their "-" marker on the same level as the parent. This bad reformatting is done all throughout this file.